src/connectors/voice-search/__tests__/connectVoiceSearch-test.js

```diff
@@ -65,6 +65,28 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/voice-searc
         })
       );
     });
+
+    it('creates custom voice helper', () => {
+      const voiceHelper = {
+        isBrowserSupported: () => true,
+        dispose: () => {},
+        getState: () => ({
+          isSpeechFinal: true,
```

Comment:
Does this state translate to other voice solutions? Can you copy/paste a usage with another solution so that we better understand how the API can work?

Reply:
other api didn't need isSpeechFinal since it didn't give intermediary results (only will transcribe once text is finished, and thus isSpeechFinal will be true). A fully local transcribing service would be able to use the variable though

Reply:
Do you have an external API implementation usage so we better understand?

Reply:
it's not yet public, so I sent it on slack